presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2557,6 +2557,41 @@ public void testCurrentUserInView()
         assertUpdate("DROP TABLE test_accounts");
     }
 
+    @Test
+    public void testCollectColumnStatisticsOnWriteSwitches()
+    {
+        assertCollectColumnStatisticsOnWrite(false);
```

Comment:
do we normally have tests that our session properties are actually gating the things we think they do?  The test is fine, but curious why for this?

Reply:
Before that there was more complicated logic involving both, table and session property. Since now the logic is trivial i will just remove this test.